### PR TITLE
Update weakref.rst

### PR DESCRIPTION
**Pull request by geryogam**

### CODE REVIEW

File: Doc/library/weakref.rst [hunk 1]
```diff
@@ -65,8 +65,8 @@ exposed by the :mod:`weakref` module for the benefit of advanced uses.
 
 Not all objects can be weakly referenced; those objects which can include class
 instances, functions written in Python (but not in C), instance methods, sets,
-frozensets, some :term:`file objects <file object>`, :term:`generator`\s, type
-objects, sockets, arrays, deques, regular expression pattern objects, and code
+frozensets, some :term:`file objects <file object>`, :term:`generators <generator>`,
+type objects, sockets, arrays, deques, regular expression pattern objects, and code
```
Comment (mangrisano) [inline Doc/library/weakref.rst:69]: I think it would be better remove the comma before "and".

**Review by mangrisano — CHANGES_REQUESTED** [review]

**Comment by mangrisano** [comment]
Hi, thank you for the pull request.

I've left a suggestion about the change.

Thank you. :)

File: Doc/library/weakref.rst (hunk 1 above)
Comment (csabella) [reply Doc/library/weakref.rst:69]: Actually,[ the comma before the `and`](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) is correct and just a matter of style.

**Review by csabella — APPROVED** [review]
LGTM, thanks!

**Comment by csabella** [comment]
Thanks for the PR @maggyero  and for the review @mangrisano.  :tada:

**Comment by maggyero** [comment]
Thank you for the review @csabella and @mangrisano.

**Pull request merged**